gpt4all-bindings/python/docs/gpt4all_python_embedding.md

```diff
@@ -1,18 +1,20 @@
 # Embeddings
-GPT4All supports generating high quality embeddings of arbitrary length documents of text using a CPU optimized contrastively trained [Sentence Transformer](https://www.sbert.net/). These embeddings are comparable in quality for many tasks with OpenAI.
+GPT4All supports generating high quality embeddings of arbitrary length text using any embedding model supported by llama.cpp. These include:
```

Comment:
You should reference Nomic's official guide on embeddings which exists here: https://docs.nomic.ai/atlas/guides/embeddings

Reply:
Use this line: `To learn more about embeddings read Nomic's Official Embeddings Guide.`

Reply:
That guide seems too Atlas-specific to recommend in the introductory paragraph, but I like its section on task types.